pkg/planner/core/rule_predicate_push_down.go

```diff
@@ -385,6 +385,8 @@ func (p *LogicalJoin) getProj(idx int) *LogicalProjection {
 
 // simplifyOuterJoin transforms "LeftOuterJoin/RightOuterJoin" to "InnerJoin" if possible.
 func simplifyOuterJoin(p *LogicalJoin, predicates []expression.Expression) {
+	return
```

Comment:
is this return expected？

Reply:
Not intended to be reviewed yet  :) Just put this to disable old code path.